benches/benches/transaction_throughput.rs

```diff
@@ -90,6 +93,12 @@ where
             test_builder.utxo_validation = true;
             test_builder.gas_limit = Some(10_000_000_000);
             test_builder.block_size_limit = Some(1_000_000_000_000);
+            test_builder.database_type = DbType::RocksDb;
+            test_builder.database_config = DatabaseConfig {
+                cache_capacity: Some(16 * 1024 * 1024 * 1024),
+                max_fds: -1,
+                columns_policy: Default::default(),
```

Comment:
Can we use `OnCreation` here explicitly, please? Since it is crucial for this test=) If someone change default implementation, the test will receive regression

Reply:
DOne.